doc/deployment/amazon_web_services.md

```diff
@@ -1,30 +1,90 @@
 # Amazon Web Services
 
+## Prerequisites
+
+- [AWS CLI](https://aws.amazon.com/cli/) - have it installed and have your [AWS credentials](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) configured.
+- [kubectl](https://kubernetes.io/docs/user-guide/prereqs/)
+- [kops](https://github.com/kubernetes/kops/blob/master/docs/install.md)
+
 Below, we show how to deploy Pachyderm on AWS in a couple of different ways:
 
-1. [By manually deploying Kubernetes and Pachyderm.](amazon_web_services.html#manual-pachyderm-deploy)
-2. [By executing a one shot deploy script that will both deploy Kubernetes and Pachyderm.](amazon_web_services.html#one-shot-script)
+1. [By executing a one shot deploy script that will both deploy Kubernetes and Pachyderm.](amazon_web_services.html#one-shot-script)
```

Comment:
We discussed this in the past and decided to make the manual deploy the first/preferred route in the docs. I'm not opposed to changing it, but we have received feedback both ways. There was a time that I had the one shot script first, but we got a bunch of feedback that people weren't able to change or customize what they wanted.  Also, people were complaining that the one shot script deployed both k8s and pachyderm, when they already had a k8s they wanted to use. 

Anyway, I can see both sides. Maybe @sjezewski has some opinions on this as well.

Reply:
Maybe an alternative here would be to better emphasize at the beginning of the docs who should use which deploys and why?

Reply:
I think I mentioned this in person too, but I think the best solution here may to go with just emphasizing which options are better for which setups instead of changing the order.

Reply:
That sounds good. I added a section at the top called "options" that explains the two paths. The reason I started editing this doc is that the customer I was working with missed this option, so I think some added emphasis will avoid that going forward